docs/core/diagnostics/sample-debug-target.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Sample debug target
+description: Introduction to the sample debug target used in the scenarios.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 10/14/2019
+---
+# Sample debug target
```

Comment:
Would it better to call this "Sample .NET Core diagnostics app" or something like that?

Reply:
`sample debug target` does sound a bit too formal.  I don't like "Sample .NET Core diagnostics app" it doesn't convey the bad examples message. I did a quick look for common terminology, but didn't find anything. I am tempted to keep it as is.